src/components/SearchQuery/hits.jsx

```diff
@@ -2,75 +2,132 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connectHits, Highlight } from 'react-instantsearch-dom';
 
+function RefHighlight(props) {
+  if (props.link) {
+    return (
+      <a href={props.link} target="_blank" rel="noreferrer" >
+        <Highlight attribute={props.attribute} hit={props.hit} />
+      </a>
+    );
+  }
+  return <Highlight attribute={props.attribute} hit={props.hit} />;
+}
+
+RefHighlight.propTypes = {
+  link: PropTypes.string,
+  attribute: PropTypes.string,
+  hit: PropTypes.shape().isRequired,
+};
+RefHighlight.defaultProps = {
+  link: '',
+  attribute: '',
+};
+
+function FormatHit(props) {
+  return (
+    <span>
+      <strong> {props.title}: </strong>
```

Comment:
This renders a bit strangely for ballot measures. If you search the term "measure", you'll see it shows as " - : ", as in "Measure: QQ - : Youth Voting" and "Measure: Y - : OUSD Facilities Bond".